ios/CameraQueues.swift

```diff
@@ -24,6 +24,13 @@ public class CameraQueues: NSObject {
                                                      autoreleaseFrequency: .inherit,
                                                      target: nil)
```

Comment:
do we still need that queue then?

Reply:
Will need to check as part of the issue with the synchronizer - it can probably be removed as we are still getting the `Depth data out of sync` issue with it on this separate queue anyways 😅

Reply:
Queue removed - not required for keeping them in sync and produced the same error with/without it